Tasks/AzureRmWebAppDeployment/make.json

```diff
@@ -11,6 +11,11 @@
                 "archiveName": "MSDeploy.zip",
                 "url": "https://vstsagenttools.blob.core.windows.net/tools/MSDeploy/3.6/MSDeploy.zip",
                 "dest": "./"
+            },
+            {
+                "archiveName": "ctt.zip",
+                "url": "https://vstsagenttools.blob.core.windows.net/tools/ctt/1.6/ctt.zip",
```

Comment:
I am fine with this change. Have you synced your branch with rest of the changes from variable substitution. If not, please do so and publish another iteration for review.